cmd/mirror-url.go

```diff
@@ -126,6 +126,19 @@ func deltaSourceTarget(ctx context.Context, sourceURL, targetURL string, opts mi
 		URLsCh <- URLs{Error: err.Trace(targetAlias, targetURL)}
 		return
 	}
+	// It will change the expanded alias back to the alias
+	// again, by replacing the sourceUrlFull with the sourceAlias.
+	// This url will be used to mirror.
+	sourceAlias, sourceURLFull, _ := mustExpandAlias(sourceURL)
```

Comment:
This is already done in L113

Reply:
Ah yes. Will change to use the already got value.